src/plugins/discover/public/application/angular/discover.js

```diff
@@ -28,31 +27,49 @@ import { getState, splitState } from './discover_state';
 
 import { RequestAdapter } from '../../../../inspector/public';
 import {
+  connectToQueryState,
   esFilters,
   indexPatterns as indexPatternsUtils,
-  connectToQueryState,
   syncQueryStateWithUrl,
 } from '../../../../data/public';
-import { SavedObjectSaveModal, showSaveModal } from '../../../../saved_objects/public';
-import { getSortArray, getSortForSearchSource } from './doc_table';
+import { getSortArray } from './doc_table';
 import { createFixedScroll } from './directives/fixed_scroll';
 import * as columnActions from './doc_table/actions/columns';
 import indexTemplateLegacy from './discover_legacy.html';
-import { showOpenSearchPanel } from '../components/top_nav/show_open_search_panel';
 import { addHelpMenuToAppChrome } from '../components/help_menu/help_menu_util';
 import { discoverResponseHandler } from './response_handler';
 import {
+  getAngularModule,
+  getHeaderActionMenuMounter,
   getRequestInspectorStats,
   getResponseInspectorStats,
   getServices,
-  getHeaderActionMenuMounter,
   getUrlTracker,
-  unhashUrl,
+  redirectWhenMissing,
   subscribeWithScope,
   tabifyAggResponse,
-  getAngularModule,
-  redirectWhenMissing,
 } from '../../kibana_services';
+import {
+  getRootBreadcrumbs,
+  getSavedSearchBreadcrumbs,
+  setBreadcrumbsTitle,
+} from '../helpers/breadcrumbs';
+import { validateTimeRange } from '../helpers/validate_time_range';
+import { popularizeField } from '../helpers/popularize_field';
+import { getSwitchIndexPatternAppState } from '../helpers/get_switch_index_pattern_app_state';
+import { addFatalError } from '../../../../kibana_legacy/public';
+import {
+  DEFAULT_COLUMNS_SETTING,
+  MODIFY_COLUMNS_ON_SWITCH,
+  SAMPLE_SIZE_SETTING,
+  SEARCH_ON_PAGE_LOAD_SETTING,
+} from '../../../common';
+import { resolveIndexPattern, loadIndexPattern } from '../helpers/resolve_index_pattern';
+import { getTopNavLinks } from '../components/top_nav/get_top_nav_links';
+import { updateSearchSource } from '../helpers/update_search_source';
+import { calcFieldCounts } from '../helpers/calc_field_counts';
+
+const services = getServices();
```

Comment:
Note that the IDE did some of this reorderings, import before variable declaration, sort import alphabetical, kept it because they make sense